### ISSUE TITLE
403 Forbidden for api/annotation/allocation/info at startup

### ISSUE THREAD

**Issue by IllDepence** [allenai/pawls#135]
Hi,

when I run `$ docker-compose up --build` all containers appear to be starting and running. But when I then point my browser at `localhost:8080` I get a loading icon and nothing happens. When I reload the page with the browser developer tools open I get an error message *"Unhandled Rejection (Error): Request failed with status code 403"*.

Looking at the network traffic in the browser developer tools it appears that an access of `api/annotation/allocation/info` results in a 403 response.

Complete console output and website with error message attached.

[console_output.txt](https://github.com/allenai/pawls/files/6925228/console_output.txt)
[trace.html.txt](https://github.com/allenai/pawls/files/6925223/trace.html.txt)

Any idea what's going wrong in this case?

Thanks in advance for any help.

**Comment by codeviking** [comment]
Hi @IllDepence,

I'm unable to reproduce the issue you're seeing. When I follow the instructions and start things up, I see this:

![Screen Shot 2021-08-03 at 3 58 32 PM](https://user-images.githubusercontent.com/801451/128096862-c3254c8b-ad5d-4316-9840-14480e64fff7.png)

This isn't surprising as in this scenario I haven't downloaded any PDFs to annotate.

Can you pull the latest code and see if you're still running into the issue? Depending on how you have things checked out you might have to do something like:

```
git fetch origin
git pull origin main
```

Of note, somewhat recently we switched away from using the `master` branch in favor of using `main`. It could be that you're still pulling from the former.

Let us know.  Thanks!

**Comment by IllDepence** [comment]
Hi @codeviking,

thank you for the quick reply. An outdated state of the repo was not the problem, but I've got it working now.

Regarding the problem I described above, I'm not sure what caused it in the end but maybe it was due to me messing something up with the `skiff_files` directory. Because in the README is says “*PDFs should be put in `skiff_files/apps/pawls`.*” but `skiff_files` in the repository has no subdirectories, I created the subdirectories myself and just placed some PDFs inside.

In any case, I started with a fresh copy of PAWLS and just running `docker-compose up --build` gave me the output you show above.

Through the process of then getting local PDFs to work I then stumbled at three points:

1. This will likely depend on a system's docker setup, but running `docker-compose up --build` on my system created the directory `skiff_files/apps/pawls/` as the root user. `pawls preprocess` run as a normal user then wasn't able to write into the PDFs directories. (fixed w/ chown)
2. Because I now had this `skiff_files/apps/pawls/` directory created for me, and because I knew about the `skiff_files/apps/pawls` volume in `docker-compose.yaml` it took me some time to notice that PDFs have to be placed in (individual subdirectories within) *a subdirectory named `papers`* within `skiff_files/apps/pawls/`
3. Running `pawls assign skiff_files/apps/pawls/papers development_user --all` gives me the error
    `Error: Provided annotator was not a valid email.`
    instead running `pawls assign skiff_files/apps/pawls/papers 'development_user@example.com' --all` works.
